cmd/operator-sdk/run/local.go

```diff
@@ -65,6 +66,8 @@ func (c *runLocalArgs) addToFlags(fs *pflag.FlagSet) {
 func (c runLocalArgs) run() error {
 	log.Infof("Running the operator locally in namespace %s.", c.watchNamespace)
 
+	projutil.MustInProjectRoot()
```

Comment:
it was been called inside of each run. It is a required check for all types.

Reply:
This can actually be removed from here I think. We call this first thing in the `operator-sdk run` command here: https://github.com/operator-framework/operator-sdk/blob/48b720395368cec82a9e2f2530d3d460557bb216/cmd/operator-sdk/run/cmd.go#L73

Reply:
Verified. Yes, you are right, we can remove it.